src/umpire/resource/HostSharedMemoryResource.cpp

```diff
@@ -45,15 +49,35 @@ void HostSharedMemoryResource::deallocate(void* ptr, std::size_t)
   return pimpl->deallocate(ptr);
 }
 
-bool HostSharedMemoryResource::isAccessibleFrom(Platform p) noexcept
+bool HostSharedMemoryResource::isPageable() noexcept
 {
-  // Todo: Need to determine how to update tests which require
-  // unnamed alloc to test.
-  //
-  UMPIRE_USE_VAR(p);
+#if defined(UMPIRE_ENABLE_CUDA)
+  int pageableMem = 0;
+  int cdev = 0;
+  cudaGetDevice(&cdev);
+
+  //Device supports coherently accessing pageable memory
+  //without calling cudaHostRegister on it
+  cudaDeviceGetAttribute(&pageableMem,
+     cudaDevAttrPageableMemoryAccess, cdev);
+  if(pageableMem)
+    return true;
+#endif
+  // Note: Regarding omp_target, we pick a default of false here 
```

Comment:
Is this comment located in the right spot? Like, if we're on cuda platform, would we actually get here if we were also dealing with omp_target?